### PR TITLE
Integrated Katacoda tutorials in explore section

### PR DESCRIPTION
**Pull request by GuentherJulian**

### CODE REVIEW

File: website/pages/explore/dir-content/cobigen-tutorial.asciidoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+= COBIGEN TUTORIAL
```
Comment (maybeec) [inline website/pages/explore/dir-content/cobigen-tutorial.asciidoc]: I think we should not introduce additional nodes but simply add the links to the already available nodes like cobigen.
Same for the other nodes

**Review by maybeec — CHANGES_REQUESTED** [review]

File: website/pages/explore/dir-content/cobigen-tutorial.asciidoc (hunk 1 above)
Comment (GuentherJulian) [reply website/pages/explore/dir-content/cobigen-tutorial.asciidoc]: I've integrated the kafka and cobigen tutorials. What do you think is the best place for the jump-the-queue and my-thai-star links?

Comment (maybeec) [reply website/pages/explore/dir-content/cobigen-tutorial.asciidoc]: Both are samples, maybe we should add a new samples folder and add mythaistar and jump the queue in Teo different nodes where we could also link to the github repo and docs sections.

Comment (GuentherJulian) [reply website/pages/explore/dir-content/cobigen-tutorial.asciidoc]: I have created now a new samples node with two separate nodes for both of them.

**Pull request merged**